.github/workflows/ci-release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release CI
+on:
+  push:
+    tags:
+      - 'release/v?[0-9]*'
+
+env:
+  RELEASE_TAG: ${{ github.ref_name }}
+
+jobs:
+  publish:
+    name: Publish to Packagist
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          path: main
+      - name: Set up environment variables
+        run: ./.github/scripts/publish_env.sh >> $GITHUB_ENV
+        working-directory: ./main
+        shell: bash
+      - name: Install PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.3"
+          tools: composer:v2
+      - name: Pack API client
+        run: ./.github/scripts/pack.sh
+        working-directory: ./main
+        shell: bash
+      - name: Create tag for Packagist (Dry run)
+        if: ${{ env.DRY_RUN == '1' }}
```

Comment:
**nit**: I tend to put `if:` as the first line of a step so it's immediately clear that the step is conditional.

Reply:
I think that's a matter of preference... I like to have the name first so I know what it's called
Also has the advantage of being possible to add or remove the conditional without modifying additional lines.

If we want to standardize on a coding standard though, I'm up for that.